App changes for ETD

jamesvanmil opened the pull request:
Closes #387

hortongn commented:
You have three files you've changed in Curate, but have not made the same changes to the existing files in Scholar:
* config/college_and_department.yml
* config/initializers/curate_config.rb
* config/locales/sufia.en.yml

jamesvanmil commented:
I've addressed the `college_and_department.yml` and the `curate_config.rb` (on the latter we missed  `:student_work`, so I added that too).

`sufia.em.yml` is good without replication.
